pages/common/find.md

```diff
@@ -33,3 +33,7 @@
 - Find files matching more than one search criteria:
 
 `find {{root_path}} -name '{{*.py}}' -or -name '{{*.r}}'`
+
+- Find files matching path pattern while excluding some certain path:
+
+`find {{root_path}} -name '{{*.py}}' -not -path '*/site-packages/*'`
```

Comment:
Sorry, I can't understand the meaning of tokenize.  
I imitatethe structure of `find {{root_path}} -name '{{*.py}}' -or -name '{{*.r}}'` from `find.md`